Add com.evepreview.manager

h0lylag opened the pull request:
<!-- ⚠️⚠️  Submission pull request MUST be made against the `new-pr` **base branch** ⚠️⚠️  -->

<!-- 💡 Please go to the preview tab to view the markdown below 💡 -->

### Please confirm your submission meets all the criteria

<!-- 💡 Please replace each `[ ]` with `[X]` when the step is complete 💡 -->

<!-- 💡 Please tick and write 'N/A' with a reason if a checklist item below is not applicable 💡 -->

- [X] Please describe the application briefly. EVE Preview Manager helps players of the MMO EVE Online multibox multiple clients. It is a TOS compliant player aid. It shows live previews of multiple clients and allows the player to use hotkeys to switch between them quickly. 
- [X] Please attach a video showcasing the application on Linux using the Flatpak. https://www.youtube.com/watch?v=Hgdb67pHMVY
- [X] The Flatpak ID follows all the rules listed in the [Application ID requirements][appid].
- [X] I have read and followed all the [Submission requirements][reqs] and the [Submission guide][reqs2] and I agree to them.
- [X] I am an author to the project.

<!-- 💡 Please mention below the GitHub usernames of any additional maintainers needed (if any) 💡 -->

<!-- ⚠️⚠️  Please DO NOT modify anything below this line ⚠️⚠️  -->

[appid]: https://docs.flathub.org/docs/for-app-authors/requirements#application-id
[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[reqs2]: https://docs.flathub.org/docs/for-app-authors/submission

hfiguiere commented on assets/com.evepreview.manager.png:
you can remove this it's not necessary. SVG is enough

hfiguiere commented on com.evepreview.manager.metainfo.xml:
```diff
@@ -0,0 +1,84 @@
+
+  <screenshots>
+    <screenshot>
+      <image>https://i.imgur.com/k5BYO7v.png</image>
```
imgur is completely inappropriate for screenshot. it will break.

h0lylag replied:
I have updated the screenshots to be hosted on my server where evepreview.com is hosted, and updated the now upstreamed metainfo to reflect this.

h0lylag commented:
> The domain to be used for verification is evepreview.com. If you intend to verify this app, please confirm that you can upload https://evepreview.com/.well-known/org.flathub.VerifiedApps.txt. Otherwise, ignore this. Please comment if this incorrect.

I have uploaded a org.flathub.VerifiedApps.txt to the expected location. Currently it is without verification code. But it is ready to be updated.


> This file should be (submitted and managed) upstream.

I have removed from this PR the SVG, metainfo, and desktop files and moved them upstream. 


> you can remove this it's not necessary. SVG is enough

The png has been removed.


> imgur is completely inappropriate for screenshot. it will break.

I have updated the screenshots to be hosted on my server where evepreview.com is hosted, and updated the now upstreamed metainfo to reflect this.


Additionally, I am requesting an exception for the `finish-args-own-name-wildcard-org.kde` error. Currently, the program needs this for its tray icon to work.

h0lylag commented on assets/com.evepreview.manager.svg:
This file has been removed from this PR and moved to the upstream project.

h0lylag commented on com.evepreview.manager.metainfo.xml:
This file has been removed from this PR and moved to the upstream project.

h0lylag commented:
bot, build

h0lylag commented:
Hello,

I believe I have addressed the 5 issues hfiguiere identified. If not, please let me know so I can make any needed updates.

In regards to the exception I was requesting earlier in https://github.com/flathub-infra/flatpak-builder-lint/pull/801, as bbhtt directed, I have fixed the upstream project to no longer need this.

Thank you for your time.
